package.json

```diff
@@ -183,7 +183,7 @@
         "electron-updater": "^4.6.5",
         "idb-keyval": "^6.1.0",
         "lodash": "^4.17.21",
-        "luxon": "^2.3.0",
```

Comment:
I am surprised to see that luxon was bumped as well, dependabot usually only updates a single dependency in package.json.

I think I'll make dependabot recreate this PR.

Reply:
Hrm, maybe I'll try a rebase first since luxon was bumped in https://github.com/microsoft/accessibility-insights-web/pull/5203/files

Reply:
A rebase worked! Only `simple-git` is in the diff now.